Shared/Samples/Take screenshot/TakeScreenshotView.swift

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct TakeScreenshotView: View {
+    /// The current draw status of the map.
+    @State private var currentDrawStatus: DrawStatus = .inProgress
+    /// The image to export.
+    @State private var imageToExport: UIImage?
+    /// The map with an imagery basemap centered on Hawaii.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISImageryStandard)
+        map.initialViewpoint = Viewpoint(
+            latitude: 20.78,
+            longitude: -156.84,
+            scale: 5_000_000
+        )
+        return map
+    }()
+    
+    /// A Boolean value indicating whether the screenshot view is presented.
+    var screenshotViewIsPresented: Binding<Bool> {
+        Binding {
+            imageToExport != nil
+        } set: { newValue in
+            if !newValue {
+                imageToExport = nil
+            }
+        }
+    }
```

Comment:
This is a bit hacky. Instead, consider making `Screenshot` adopt `Identifiable` (by adding a `let id = UUID()` property to it) and then using the screenshot as the item for the sheet:

```swift
.sheet(item: $screenshot) { screenshot in
    ...
}
```